src/index.js

```diff
@@ -0,0 +1,137 @@
+// wave 2 change temperature
+const tempValue = document.getElementById('tempValue');
+const tempIncrease = document.getElementById('increaseTempControl');
+const tempDecrease = document.getElementById('decreaseTempControl');
+const landscape = document.getElementById('landscapeDisplay');
```

Comment:
It's descriptive and makes your code self-documenting when your variable names also indicate what the element is. 
```suggestion
const tempValueDisplay = document.getElementById('tempValue');
const tempIncreaseButton = document.getElementById('increaseTempControl');
const tempDecreaseButton = document.getElementById('decreaseTempControl');
const landscapeDisplay= document.getElementById('landscapeDisplay');
```